### PR TITLE
[fix](statistics)Skip analyze if the collected info is invalid.

### PR DESCRIPTION
**Pull request by Jibing-Li**
To handle slow compaction and unknown bug, we need to eliminate invalid stats info. For example, ndv>rowCount,  ndv is 0 but min/max exists and so on.

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java [hunk 1]
```diff
@@ -484,6 +484,11 @@ protected void runQuery(String sql) {
         try (AutoCloseConnectContext a  = StatisticsUtil.buildConnectContext(false)) {
             stmtExecutor = new StmtExecutor(a.connectContext, sql);
             ColStatsData colStatsData = new ColStatsData(stmtExecutor.executeInternalQuery().get(0));
+            if (!colStatsData.isValid()) {
+                String message = String.format("ColStatsData is invalid, skip analyzing. %s", colStatsData.toSQL(true));
+                LOG.info(message);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java:489]: warning? could we get table column and invalid data from log message?

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java [hunk 2]
```diff
@@ -185,4 +185,24 @@ public ColumnStatistic toColumnStatistic() {
             return ColumnStatistic.UNKNOWN;
         }
     }
+
+    public boolean isNull(String value) {
+        return value == null || value.equalsIgnoreCase("NULL");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java:190]: why need `value.equalsIgnoreCase("NULL")` ?

File: fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java (hunk 1 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java:489]: Fixed. We could get column name, table, db and catalog id in the log message.

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java (hunk 2 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java:190]: Will fix it in a separate pr.

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run external

**Review by morrySnow — APPROVED** [review]

**Review by xzj7019 — APPROVED** [review]

**Pull request merged**